Sources/DashdevsNetworking/Classes/RequestDescriptor.swift

```diff
@@ -21,6 +21,7 @@ public protocol RequestDescriptor {
     var responseError: Deserializator<ResourceError>? { get }
     var parameters: BodyParameters? { get }
     var versionPath: Path? { get }
+    var overrideURL: URL? { get }
```

Comment:
nit: I'd use baseURL name and add a comment that default one if used if that's one is nil